pkg/common/fflag/fflag.go

```diff
@@ -91,6 +91,23 @@ func Load(rc RawConfig) error {
 	return nil
 }
 
+// Unload resets the feature flags states to its default values.
```

Comment:
I think that we need to update the comment for the Load function, to reflect the fact that Unload can be called to reset the state. It would be probably a good idea to point out that the Unload function is only designed for testing, and it's not expected to be called by the normal execution of SPIRE.